components/common/src/command/package/install.rs

```diff
@@ -348,7 +348,7 @@ where
 
     let channel = match channel {
         Some(c) => Cow::Borrowed(c),
-        None => Cow::Owned(ChannelIdent::default()),
+        None => Cow::Owned(ChannelIdent::default()), // or configured_value?
```

Comment:
`default` is good here, though I think we might actually be able to remove the `Option`ality of this parameter now; I don't see any places where we're passing `None`.

Reply:
Ooh, good call. Will do!